### PR TITLE
add previous and next relations

### PR DESCRIPTION
**Pull request by xenoterracide**
GH #51 

https://www.w3.org/TR/html5/links.html#link-type-next

### CODE REVIEW

File: layouts/_default/list.html [hunk 1]
```diff
@@ -37,16 +37,16 @@
...
-  <a class="btn btn-outline-primary" href="{{ .Paginator.Prev.URL }}" role="button">{{ i18n "paginatorPrevious" }}</a>
-  <a class="btn btn-outline-primary" href="{{ .Paginator.Next.URL }}" role="button">{{ i18n "paginatorNext" }}</a>
+  <a class="btn btn-outline-primary" href="{{ .Paginator.Prev.URL }}" rel="prev" role="button">{{ i18n "paginatorPrevious" }}</a>
+  <a class="btn btn-outline-primary" href="{{ .Paginator.Next.URL }}" rel="next" role="button">{{ i18n "paginatorNext" }}</a>
   {{ end }}
   {{ if and (.Paginator.HasPrev) (not .Paginator.HasNext) }}
-  <a class="btn btn-outline-primary" href="{{ .Paginator.Prev.URL }}" role="button">{{ i18n "paginatorPrevious" }}</a>
-  <a class="btn btn-outline-primary disabled" href="#" role="button" aria-disabled="true">{{ i18n "paginatorNext" }}</a>
+  <a class="btn btn-outline-primary" href="{{ .Paginator.Prev.URL }}" rel="prev" role="button">{{ i18n "paginatorPrevious" }}</a>
+  <a class="btn btn-outline-primary disabled" href="#" role="button" rel="next" aria-disabled="true">{{ i18n "paginatorNext" }}</a>
```
Comment (alanorth) [inline layouts/_default/list.html]: In the case there is no next page we shouldn't use `rel="next"`.

File: layouts/_default/list.html [hunk 2]
```diff
@@ -37,16 +37,16 @@
...
   {{ end }}
   {{ if and (not .Paginator.HasPrev) (.Paginator.HasNext) }}
-  <a class="btn btn-outline-primary disabled" href="#" role="button" aria-disabled="true">{{ i18n "paginatorPrevious" }}</a>
-  <a class="btn btn-outline-primary" href="{{ .Paginator.Next.URL }}" role="button">{{ i18n "paginatorNext" }}</a>
+  <a class="btn btn-outline-primary disabled" href="#" role="button" rel="prev" aria-disabled="true">{{ i18n "paginatorPrevious" }}</a>
```
Comment (alanorth) [inline layouts/_default/list.html]: In the case there is no previous page we shouldn't use `rel="prev"`.

**Review by alanorth — CHANGES_REQUESTED** [review]
Good addition, but need two minor changes.

**Comment by alanorth** [comment]
Psst, @xenoterracide. Two small changes to get this merged.

**Comment by xenoterracide** [comment]
done,  I can squash if necessary though I don't think I can do that on GH

**Comment by alanorth** [comment]
Looks good to me, @xenoterracide. Don't worry about squashing, I'm not Linus Torvalds so my standards for commits are slightly lower—at least you didn't add a bunch of unnecessary whitespace! 😝

**Comment by alanorth** [comment]
Oh, looks like I squashed it into one commit on the GitHub interface. Pretty slick!

**Pull request merged**